packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,64 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
+
+- (NSURL * _Nullable)devLauncherURL
+{
+#ifdef EX_DEV_LAUNCHER_URL
+  NSString *urlString = [NSString stringWithFormat:@"%s/%s", EX_DEV_LAUNCHER_URL, EX_DEV_LAUNCHER_PACKAGER_PATH];
+  return [NSURL URLWithString:urlString];
+#endif
+  return nil;
+}
+
+- (NSURL * _Nullable)devLauncherStatusURL
+{
+#ifdef EX_DEV_LAUNCHER_URL
+  NSString *urlString = [NSString stringWithFormat:@"%s/status", EX_DEV_LAUNCHER_URL];
+  return [NSURL URLWithString:urlString];
+#endif
+  return nil;
+}
+
+- (BOOL)isLauncherPackagerRunning
```

Comment:
Should we put in an assertion that this code is running off of the main thread/main queue? The reason I ask is because of the blocking NSURLSession task below.

Reply:
Done.